src/main/java/com/iridium/iridiumskyblock/managers/IslandManager.java

```diff
@@ -101,10 +101,11 @@ public void setIslandBiome(@NotNull Island island, @NotNull XBiome xBiome) {
         getIslandChunks(island, world).thenAccept(chunks -> {
             Location pos1 = island.getPos1(world);
             Location pos2 = island.getPos2(world);
-            xBiome.setBiome(pos1, pos2);
-            for (Chunk chunk : chunks) {
-                IridiumSkyblock.getInstance().getNms().sendChunk(world.getPlayers(), chunk);
-            }
+            xBiome.setBiome(pos1, pos2).thenAccept(unused -> {
```

Comment:
use .thenRun instead of thenAccept

Reply:
Sure? XSeries runs async

Reply:
? wdym thenRun just removes the unused variable